example_test.go

```diff
@@ -34,13 +33,13 @@ func ExampleRsync() {
 
 	go func() {
 		defer close(opsOut)
-		rs.CreateDelta(currentReader, sig, writeOperation)
+		_ = rs.CreateDelta(currentReader, sig, writeOperation)
 	}()
 
 	var newWriter strings.Builder
-	oldReader.Seek(0, os.SEEK_SET)
+	_, _ = oldReader.Seek(0, io.SeekStart)
```

Comment:
```suggestion
	_, _ = oldReader.Seek(0, io.SeekStart)
```

Can you make sure to only send just this change? all other changes are not needed.